storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/prometheus/prometheus/model/value"
 	"github.com/prometheus/prometheus/prompb"
 
-	prometheustranslator "github.com/prometheus/prometheus/storage/remote/otlptranslator/prometheus"
+	prometheustranslator "github.com/prometheus/otlp-prometheus-translator"
```

Comment:
The linter should have caught it, but this import should be be moved to the block starting at line 30, i.e. non-stdlib/non-local imports.